### PR TITLE
Colliding with Quarkus

### PR DESCRIPTION
**Pull request by ggotti**
Blog post about the new Cloud-First Java Framework from RedHat.

### CODE REVIEW

File: posts/colliding-with-quarkus.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## GraalVM
+[GraalVM](https://www.graalvm.org/) is a relatively new universal virtual machine for running JVM-based languages, as well as JavaScript, Python, Ruby and R. The feature that Quarkus really embraces is the [SubstrateVM](https://github.com/oracle/graal/tree/master/substratevm), which allows Java applications to be compiled into native executable images. Instead of running your Java code against a JVM, you can compile it down to a portable, native executable. Quarkus abstracts all this complexity from you, and allows you to just run `nativeImage` to build a executable binary.
+
+# Taking it for a spin
+So I have put together a [small HTTP server example](https://github.com/ggotti/kanye-quarkus) which basically just proxies requests to the `https://api.kanye.rest` service, which returns a random Kanye West quotes. The application has three small classes:
```
Comment (dsharpe) [inline posts/colliding-with-quarkus.md]: either remove "a" or change "quotes" to "quote"

File: posts/colliding-with-quarkus.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- VoteKanyeResource
+- KanyeService
+- KanyeQutote
```
Comment (dsharpe) [inline posts/colliding-with-quarkus.md]: typo?

Comment (ggotti) [reply posts/colliding-with-quarkus.md]: 👍

File: posts/colliding-with-quarkus.md (hunk 1 above)
Comment (ggotti) [reply posts/colliding-with-quarkus.md]: 👍

**Pull request merged**